tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml

```diff
@@ -12,7 +12,12 @@
 
         mkdir -p SVECTOR &&
 
-        cp $data_svector SVECTOR/data.svector &&
+        #for $ins in $data_svector:
+
+							ln -f -s  $ins SVECTOR
```

Comment:
It's a for loop and you are overwriting the SVECTOR everytime you run this look. This is a bug.